### ISSUE TITLE
[Watchmarket] Remove last_update field

### ISSUE THREAD

**Issue by EnoRage** [trustwallet/watchmarket#14]
```
{"coin":60,"token_id":"0xB8c77482e45F1F44dE1745F52C74426C631bDD52","type":"token","price":{"value":19.8148675333,"change_24h":-9.23673},"last_update":"2020-02-26T01:47:05Z"}
```
remove` last_update` from Ticker

**Comment by arein** [comment]
Why remove this?

**Comment by EnoRage** [comment]
No need at it

**Comment by EnoRage** [comment]
Solved at #101
